MSAL/test/app/ios/MSALTestAppCacheViewController.m

```diff
@@ -85,13 +93,21 @@ - (id)init
     return self;
 }
 
-- (void)deleteTokenAtPath:(NSIndexPath *)indexPath
+- (void)deleteAppMetadata:(MSIDAppMetadataCacheItem *)appMetadata
 {
-    MSIDBaseToken *token = [self tokenForPath:indexPath];
+    if (appMetadata)
+    {
+        [self.defaultAccessor removeAppMetadata:appMetadata context:nil error:nil];
```

Comment:
If removeAppMetadata is only used in test app, consider removing it from defaultAccessor and calling AccountCredentialCache directly.

Reply:
Yes it is only used in test app. Removed it from DefaultAccessor